.github/workflows/build-and-test.yml

```
@@ -48,6 +48,11 @@ jobs:
         run: | 
           pnpm build
           pnpm test
+      - name: Build and test our Lib Instrument
```

Nitpick: We can make this sound rounder by avoiding abbreviation (just like with the commons library)
```suggestion
      - name: Build and test our Instrumentation Library
```